DVOP-2122 v10 centos:8 to amazonlinux:2

autmwatl opened the pull request:
centos8 has been deprecated. migrate v10 to amazonlinux2

Commit:
DVOP-2122 v10 centos:8 to amazonlinux:2
centos8 has been deprecated. migrate v10 to amazonlinux2

sienkin commented:
The `nss` package is installed by default in Amazon Linux 2 so can be omitted from the lsit of packages.

Commit:
DVOP-2122 remove nss installation
The nss package is installed by default in Amazon Linux 2 so omit from the list of packages to install.
